packages/appstore/src/modules/traders-hub/account-with-wallets.tsx

```diff
@@ -1,115 +1,31 @@
 import React from 'react';
 import Wallet from 'Components/containers/wallet';
-import { observer, useStore } from '@deriv/stores';
+import { observer } from '@deriv/stores';
+import { useWalletList } from '@deriv/hooks';
+import { Loading } from '@deriv/components';
 
-// TODO: delete it after testing
-type TProps = {
-    show_test_wallets?: boolean;
-};
+const AccountWithWallets = observer(() => {
+    const { data, isLoading, error } = useWalletList();
+    const [selected_wallet, setSelectedWallet] = React.useState('');
```

Comment:
```suggestion
    const [selected_wallet, setSelectedWallet] = React.useState<NonNullable<typeof data>[number]>();
```
@thisyahlen-deriv The state name is `selected_wallet` so I assume we should keep the whole selected wallet data, If ID is enough then we can change it to:
```suggestion
    const [selected_wallet_id, setSelectedWalletID] = React.useState<NonNullable<typeof data>[number]['loginid']>();
```